docs/en/developer.md

```diff
@@ -423,6 +423,41 @@ for the advanced settings.
 
 ## Advanced features
 
+### Allowing email login on LDAP login form
+
+If you have registered `LDAPAuthenticator` (see "Configure SilverStripe Authenticators" section), you may notice the only
+accepted login is username on the form. This may be confusing if your users are more familiar logging in with using their email
+address.
+
+To allow logging in with email in addition to username, set the following configuration:
+
+```yaml
+LDAPAuthenticator:
+  allow_email_login: 'yes'
+```
+
+Note that your LDAP users logging in must have the `mail` attribute set, otherwise this will not work.
+
+### Falling back authentication on LDAP login form
+
+If you have users attempting to login with `LDAPAuthenticator` but still have a login in the local database
```

Comment:
It reads a bit like code, if , but, set.

You can allow users who has not been migrated to LDAP to authenticate via the default `MemberAuthenticator`.